src/main/java/net/guavy/gravestones/Gravestones.java

```diff
@@ -112,7 +113,12 @@ public static void placeGrave(World world, Vec3d pos, PlayerEntity player) {
 
 				gravestoneBlockEntity.sync();
 				block.onBreak(world, blockPos, blockState, player);
+
+				if (GravestonesConfig.getConfig().mainSettings.sendGraveCoordinates) {
+					player.sendMessage(new TranslatableText("text.gravestones.grave_coordinates", gravePos.getX(), gravePos.getY(), gravePos.getZ()), false);
+				}
 				System.out.println("[Gravestones] Gravestone spawned at: " + gravePos.getX() + ", " + gravePos.getY() + ", " + gravePos.getZ());

```

Comment:
Might be worth changing this to a logger call, it doesn't seem to actually output anything in my environment

Reply:
I'll change it on a seperate commit.